docroot/themes/humsci/humsci_colorful/theme-settings.php

```diff
@@ -37,6 +37,22 @@ function humsci_colorful_form_system_theme_settings_alter(array &$form, FormStat
     '#default_value' => theme_get_setting('theme_color_pairing'),
   ];
 
+  // Local Masthead
+  $form['options_settings']['humsci_colorful_local_masthead'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Local Masthead Settings'),
+  ];
+
+  $form['options_settings']['humsci_colorful_local_masthead']['local_masthead_variant_classname'] = [
+    '#type' => 'select',
+    '#title' => t('Local Masthead Variant'),
+    '#options' => [
+      'default' => '- Default -',
```

Comment:
Translate the string.

Reply:
Just to clarify, you want the '- Default -' string translated, correct?  Should this also be translated in the pre-existing Footer Dark mode selector then as well?

Reply:
Yeah. Any text being exposed to the user should run through translation